### ISSUE TITLE
pyenv: tab auto-completing with tab key Error "gettext command not found"

### ISSUE THREAD

**Issue by kirankotari** [pyenv/pyenv#1398]
# Describe 
I wanted to add requirements.txt file in git when I clicked on the tab key the error follows.
```
(master) λ git add repyenv: gettext: command not found                              
The `gettext' command exists in these Python versions:
  anaconda3-2019.03
pyenv: gettext: command not found
The `gettext' command exists in these Python versions:
  anaconda3-2019.03
pyenv: gettext: command not found
The `gettext' command exists in these Python versions:
  anaconda3-2019.03
pyenv: gettext: command not found
The `gettext' command exists in these Python versions:
  anaconda3-2019.03
```
- [ ] Shell: fish
- [ ] Platform information:  Darwin Mojave
- [ ] OS architecture: x86_64
- [ ] pyenv version: pyenv 1.2.13
- [ ] Python version: 
(master) λ pyenv versions
  system
  2.7.16
  3.5.6
  3.6.6
  3.6.9
* 3.7.4 (set by /Users/xyz/abc/.python-version)
  anaconda3-2019.03
  anaconda3-2019.03/envs/kaggle_manipal_env
- [ ] C Compiler information: 
Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/include/c++/4.2.1
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin

**Comment by kirankotari** [comment]
Tried adding `gettext` using `brew install gettext` and added at staring of the path. 
Still when I type `type l` it shows pyenv: gettext error

```
type l
pyenv: gettext: command not found  
The `gettext' command exists in these Python versions: 
  anaconda3-2019.03  
l# Defined in - @ line 1
```

**Comment by native-api** [comment]
Impossible to say anything without a debug trace.

To trace completion, do the following:

* Run `export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'; set -x` in the active shell
* reproduce the problem (you should get lots of tracing output and no shell prompt afterwards). Do NOT make any extraneous keypresses.
* Press `Ctrl-U` then run `set +x` to disable tracing. The shell should return to normal.

**Comment by faho** [comment]
>To trace a Bash completion, do the following:

This isn't about a bash completion, the reporter specified their shell is fish, so bash instructions don't help.

It can also be closed as it's fixed in fish 3.2, which has since been released - see https://github.com/fish-shell/fish-shell/issues/6804, and https://github.com/fish-shell/fish-shell/pull/7036.
